src/main/java/com/google/api/codegen/config/DiscoveryMethodModel.java

```diff
@@ -293,4 +293,9 @@ public TypeModel getInputType() {
   public TypeModel getOutputType() {
     return outputType;
   }
+
+  @Override
+  public String getEndpointPathTemplate() {
```

Comment:
Like my suggestion in the gax PR, I think `getPathTemplate` would work here.

Reply:
Sure thing.